data/fields/diameter_crown.json

```diff
@@ -0,0 +1,7 @@
+{
+    "key": "diameter_crown",
+    "type": "combo",
+    "label": "Crown diameter",
+    "snake_case": false,
+    "placeholder": "4 m, 9 ft, …"
```

Comment:
According to [the unit documentation](https://wiki.openstreetmap.org/wiki/Map_features/Units), the `ft` symbol is discouraged in favor of an apostrophe (`'`), for better or worse. There’s no usage of `'` or `"` so far but also hardly any usage of `ft` either. Standardizing on a single symbol for feet would facilitate the eventual introduction of a measurement field type: ideditor/schema-builder#15.